src/config-schema.json

```diff
@@ -1,218 +1,88 @@
 {
   "useEslint": {
+    "order": 1,
     "title": "ESLint Integration",
     "description":
-      "Use [prettier-eslint](https://github.com/prettier/prettier-eslint) to infer your Prettier settings based on your ESLint config. If we cannot infer a Prettier setting from your ESLint config (or if there is none in the current project), we will fallback to using your settings in the *Prettier Options* section.<br><br>**Note:** If you enable *Format on Save*, we recommend disabling ESlint's auto-fix to prevent fixing your code twice.",
+      "Use [prettier-eslint](https://github.com/prettier/prettier-eslint) to run Prettier output through ESLint's auto-fix. Automatically falls back to normal Prettier if no ESLint configuration is found.<br><br>**Note:** If you enable *Format on Save*, we recommend disabling ESlint's auto-fix to prevent fixing your code twice.",
     "type": "boolean",
-    "default": false,
-    "order": 1
+    "default": false
   },
-  "useEditorConfig": {
-    "title": "EditorConfig Integration",
+  "scopes": {
+    "order": 2,
+    "title": "Scopes on which Prettier should be run",
     "description":
-      "Use [EditorConfig](http://editorconfig.org/) configuration to load project-based settings for Prettier formatting.<br>Takes precedence over your settings. The following settings will be overriden if they are found in your `.editorconfig`: **Use tabs**, **Tab width**, **Print width**.",
```

Comment:
Maybe we should keep this option? At least until https://github.com/prettier/prettier/issues/42 is closed.
It's the third most popular request if you sort issues by 👍 reactions.

Reply:
And then do a reverse merge, like if it's in the prettier config's options, use that, if not, use the editor config's options?

Reply:
Yes, basically keep it as a fallback for people who don't use config files.